source/CommonJobs/CommonJobs.MVC.UI/Scripts/evaluations-admin.js

```diff
@@ -1,10 +1,37 @@
 ﻿$(document).ready(function () {
     var PeriodCreation = function (data) {
+        var self = this;
         this.employees = ko.observableArray();
         this.generateButtonEnable = ko.observable(false);
         if (data) {
             this.fromJS(data);
         }
+        this.headers = [
+        { title: 'Empleado', sortPropertyName: 'fullName', asc: true},
+        { title: 'Puesto', sortPropertyName: 'currentPosition', asc: true},
+        { title: 'Seniority', sortPropertyName: 'seniority', asc: true},
+        { title: 'Responsable', sortPropertyName: 'responsible', asc: true}
+        ];
+        this.activeSort;
```

Comment:
Oh, I understand now, could you assign them a default parameter to make easier to read it?